openmetadata-ui/src/main/resources/ui/src/pages/TeamsPage/TeamsPage.tsx

```diff
@@ -217,7 +217,7 @@ const TeamsPage = () => {
     setIsPageLoading(loadPage);
     try {
       const data = await getTeamByName(name, {
-        fields: 'users,parents,profile',
+        fields: 'users,parents,profile,owner',
```

Comment:
Do we need to change cypress for the same?

Reply:
No, we have unit test for the same, no need to update cypress for it.